app/schema.graphql

```diff
@@ -2059,6 +2059,7 @@ type Span implements Node {
   tokenCountTotal: Int
   tokenCountPrompt: Int
   tokenCountCompletion: Int
+  tokenPromptDetails: TokenPromptDetails!
```

Comment:
nit on the name. Can we just make it promptTokenDetails or tokenCountPromptDetaols